src/browser/api/system.js

```diff
@@ -39,7 +39,23 @@ import ofEvents from '../of_events';
 const ProcessTracker = require('../process_tracker.js');
 import route from '../../common/route';
 import { fetchAndLoadPreloadScripts } from '../preload_scripts';
-
+const verbose = 'verbose';
+const info = 'info';
+const warning = 'warning';
+const error = 'error';
+const fatal = 'fatal';
+const logLevelMappings = new Map([
```

Comment:
In spirit of KISS, why not an array with `findIndex` method to get level? I guess it's that `-1`? But was `-1` really necessary?

Reply:
In another PR, this will need to be required in another file, so this needs this to be exported

Reply:
Since these are log levels, and they will be reused elsewhere, should we put them into `log.ts`?